storage_service/locations/tests/test_wellcome.py

```diff
@@ -138,3 +141,52 @@ def set_package_to_fail(*args):
                 '/born-digital/bag.zip',
                 package=package
             )
+
+
+class TestWellcomeMoveToStorageService(TestCase):
+    fixtures = ['base.json', 'wellcome.json']
+
+    def setUp(self):
+        self.wellcome_object = models.WellcomeStorageService.objects.get(id=1)
+
+        self._s3 = boto3.client("s3", region_name='us-east-1')
+        self.tmp_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.tmp_dir)
```

Comment:
minor: If you're using `py.test` for tests (which I think we are?), the `tmp_dir` fixture is a tad cleaner: https://doc.pytest.org/en/latest/tmpdir.html

Reply:
We are, although pytest fixtures don't load so easily when `TestCase` is being used (which it is, for database fixtures)